### ISSUE TITLE
Feature Request: Excluding Chars

### ISSUE THREAD

**Issue by qwertz44** [dmuth/diceware#40]
Hi folks,
it would be great if its possible to exclude certain chars. For example "y", "z" or "@".

**Comment by dmuth** [comment]
I have no plans to do that at this time, but (like most other requests), I'll leave this one open, and if it gets a lot of upvotes, then I will adjust my priorities accordingly. :-)

**Comment by tbordovsky** [comment]
I agree it would be nice to be able to modify the SPECIAL_CHARS list for cases where passphrases do not permit specific ones.
